docker-compose-tests.yaml

```diff
@@ -20,30 +20,33 @@ services:
     build:
       context: ./database-service
       dockerfile: ./Dockerfile
-      target: unittest
+      target: integration-test
     restart: "no"
     depends_on:
       - graphql-engine
     volumes:
       - ./database-service:/app
+      - ./fixtures:/app/fixtures
     environment:
       - PYTHONDONTWRITEBYTECODE=1
```

Comment:
That's the second (and last) hack to prevent python from writing the `__pycache__` / `.pycache` folders. Because those folders are generated by the root user, they can't be deleted anymore, which is a problem for jenkins. Would we want to have those cache folders anyway, we could run the `docker-compose` command as user `id -u` as is done for fixtures generation.